src/parser.rs

```diff
@@ -362,9 +438,10 @@ impl Library {
             }
         }
 
-        if attrs.by_name("is-gtype-struct").is_some() {
-            return Ok(());
-        }
+        // TODO: unsure of how to handle this
+        //if attrs.by_name("is-gtype-struct").is_some() {
+        //    return Ok(());
+        //}
```

Comment:
Why it can't return `Ok(None)` in this case? And don't add type in `read_record_start`